test/PackageVersionsTest/InstallerTest.php

```diff
@@ -615,7 +615,7 @@ private function __construct()
      */
     public static function rootPackageName() : string
     {
-        if (!class_exists(InstalledVersions::class, false) || !InstalledVersions::getRawData()) {
+        if (!class_exists(InstalledVersions::class, false) || (method_exists(InstalledVersions::class, 'getAllRawData') && !InstalledVersions::getAllRawData()) || !InstalledVersions::getRawData()) {
```

Comment:
This will still call getRawData tho if there is data present, kinda goes like `class doesnt exist (false) || getAllRawData returns empty (false) || getRawData returns empty (false)`..

Reply:
You are correct. Should be fixed now.